src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -429,10 +429,10 @@ std::wstring Terminal::GetHyperlinkAtPosition(const COORD position)
     }
     // also look through our known pattern locations in our pattern interval tree
     const auto result = GetHyperlinkIntervalFromPosition(position);
-    if (result.value == _hyperlinkPatternId)
+    if (result.has_value() && result.value().value == _hyperlinkPatternId)
     {
-        const auto start = result.start;
-        const auto end = result.stop;
+        const auto start = result.value().start;
+        const auto end = result.value().stop;

```

Comment:
you can replace both of these `.value().` with `->`, which automatically looks inside the optional. It skips the liveness check, but this is okay because you validated that result did have a value on line 432